Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -248,5 +248,21 @@ public static void Unhover ([NotNull] this ElementScope scope)
 
       Cursor.Position = new Point (0, 0);
     }
+
+    /// <summary>
+    /// Gets an attribute from the scope or throws a <see cref="MissingHtmlException"/> if it could not be found. Wraps <see cref="ElementScope.this"/>.

```

Comment:
Maybe add an explanation why we need this extension method

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/5f2798fe450f6e8b22da92708df17b550ac141ff.